tests/api/controllers/robot_parallel_step/robot_parallel_step.c

```diff
@@ -1,3 +1,5 @@
+#include <unistd.h>
```

Comment:
Usually we list these libraries after webots and ts_* libraries.

Reply:
Fixed.